adapters/conversant/conversant.go

```diff
@@ -18,6 +19,11 @@ type ConversantAdapter struct {
 }
 
 func (c *ConversantAdapter) MakeRequests(request *openrtb2.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	cnvrRequest := *request
+	//Backend needs USD or it will reject the request
+	if cnvrRequest.Cur != nil && len(cnvrRequest.Cur) > 0 && cnvrRequest.Cur[0] != "USD" {
```

Comment:
Nitpick: There is no need for `cnvrRequest.Cur != nil` since that check is implicit with `len(cnvrRequest.Cur) > 0`. In Go, it's ok to take the length of a nil slice.

Reply:
that's easy enough I'll fix that